scripts/tag.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# Copyright 2020 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Searches for a tag in the environment or the current git branch
```

Comment:
Maybe name it `get_tag`? Otherwise it may sound like it's performing a tag action.